ledger/participant-integration-api/src/main/resources/db/migration/oracle-appendonly/V6__activate_template_id_interning.sql

```diff
@@ -0,0 +1,19 @@
+--  Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+--  SPDX-License-Identifier: Apache-2.0
+
+DROP INDEX participant_events_divulgence_template_id_idx;
```

Comment:
@meiersi-da next to dropping all the GIN indexes, we dropping the template_id single column indexes as well, since I could not find any query usage which would make use of it. What do you think?

Reply:
Yes this makes sense.